api/src/test/java/org/pmiops/workbench/api/CdrVersionsControllerTest.java

```diff
@@ -45,17 +45,16 @@
 
   private CdrVersion defaultCdrVersion;
   private CdrVersion protectedCdrVersion;
-  private DbUser user;
+  private static DbUser user;
 
   @TestConfiguration
   @Import({CdrVersionService.class, CdrVersionsController.class})
   @MockBean({FireCloudService.class})
   static class Configuration {
     @Bean
+    @Scope("prototype")
```

Comment:
Can you please put an explanation in the PR description or maybe even a small inline comment saying what this does vs the default singleton scope?

Reply:
I'm not sure we need a comment everywhere we use a language or framework feature that isn't as common as the others to describe how the feature works if it's well documented publicly. But as for *why* this one should be a prototype, I agree that would make a good comment. 

You can also use the constant `org.springframework.beans.factory.config.ConfigurableBeanFactory#SCOPE_PROTOTYPE` instead of a string literal.